frontend/src/components/profile/profile.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import ProfileMenu from "./components/profile-menu/profile-menu";
+import ProfileInfo from "./components/profile-info/profile-info";
```

Comment:
```suggestion
import { ProfileMenu, ProfileInfo } from "./components";
```
Also check **A3** criteria